control-plane/tests/test_phase37_reviewed_record_chain_rehearsal.py

```
@@ -0,0 +1,540 @@
+        execution_fixture = _require_mapping(fixture.get("execution"), "execution")
+        reconciliation_fixture = _require_mapping(
+            fixture.get("reconciliation"),
+            "reconciliation",
+        )
+        manifest = _require_mapping(fixture.get("manifest"), "manifest")
+
+        store, _ = make_store()
+        service = support.AegisOpsControlPlaneService(
+            support.RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="phase37-reviewed-shared-secret",
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)